eng/test-determinism.ps1

```diff
@@ -24,14 +22,17 @@ if ($help) {
 
 # List of binary names that should be skipped because they have a known issue that
 # makes them non-deterministic.  
-$script:skipList = @()
-
+$script:skipList = @(
+  # Added to work around https://github.com/dotnet/roslyn/issues/48417
+  "Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.dll"
```

Comment:
>"Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.dll" [](start = 2, length = 54)

The fix for parameters got merged.

Reply:
Great. I'll either get this changed here or submit a follow up PR to remove this line.